vnext/JSI.Desktop.UnitTests/ChakraCoreRuntimeUnitTests.cpp

```diff
@@ -0,0 +1,49 @@
+#include "JsiRuntimeUnitTests.h"
+
+#include "Chakra/ChakraJsiRuntimeFactory.h"
+#include "Chakra/ChakraJsiRuntimeArgs.h"
+#include "MemoryTracker.h"
+
+// TODO (yicyao): Introduces a vcxitem for shared test code and move this there.
+#include "Desktop.IntegrationTests/TestMessageQueueThread.h"
+
+#include <gtest/gtest.h>
+
+#include <vector>
+
+using facebook::jsi::JsiRuntimeUnitTests;
+using facebook::jsi::Runtime;
+using facebook::jsi::RuntimeFactory;
+using facebook::jsi::chakraruntime::makeChakraJsiRuntime;
+using facebook::jsi::chakraruntime::ChakraJsiRuntimeArgs;
+using facebook::react::CreateMemoryTracker;
+using facebook::react::MessageQueueThread;
+using Microsoft::React::Test::TestMessageQueueThread;
+
+// TODO (yicyao): We need to add tests for ChakraCoreRuntime specific behaviors
```

Comment:
>TODO (yicyao): [](start = 3, length = 14)

Please add work item id for this

Reply:
Done.

---
In reply to: [300190809](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r300190809) [](ancestors = 300190809)